kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -448,7 +467,9 @@ define([
                     });
                 })
                 .then(() => {
-                    return widget;
+                    const retValue = {};
+                    retValue[parameterId] = widget;
```

Comment:
the downside of not having sigils and allowing text not to be quoted...

Reply:
Yeah, I kinda hate that pattern. I had to use it a few times, too, and it irked me each time.
I think there's an es6-ism that might make it work as a one liner, but it would be even uglier, and I couldn't be bothered to work it out.